docker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:10
```

Comment:
We should be inheriting from the AWS official lambda container image.

There are 3 ways to do it (https://docs.aws.amazon.com/lambda/latest/dg/images-create.html)
1. [Using an AWS base image for Lambda](https://docs.aws.amazon.com/lambda/latest/dg/images-create.html#runtimes-images-lp)
2. [Using an AWS base image for custom runtimes](https://docs.aws.amazon.com/lambda/latest/dg/images-create.html#runtimes-images-custom)
3. [Using a non-AWS base image](https://docs.aws.amazon.com/lambda/latest/dg/images-create.html#images-types)

bignbit used the third option (https://github.com/podaac/bignbit/blob/develop/docker/Dockerfile) based on the example given on this page (https://docs.aws.amazon.com/lambda/latest/dg/python-image.html#python-image-clients). The reason for this was because bignbit requires gdal which need specific c libraries installed in order to work.

Because hydrocron does not use gdal and is a pure python package, it should be possible to use the first option and simply use the aws python container image as the base. I'm confused why we're starting from node here and using `npm` at all. There is is no javascript in this project, it's all python